crates/rspack_core/src/normal_module_factory.rs

```diff
@@ -806,16 +793,6 @@ impl NormalModuleFactory {
   }
 }
 
-#[derive(Debug, Clone)]
-pub struct NormalModuleFactoryContext {
-  pub original_module_identifier: Option<ModuleIdentifier>,
-  pub module_type: Option<ModuleType>,
-  pub side_effects: Option<bool>,
-  pub options: Arc<CompilerOptions>,
-  pub lazy_visit_modules: std::collections::HashSet<String>,
-  pub issuer: Option<Box<str>>,
-}
```

Comment:
`module_type`, `side_effects`, `lazy_visit_modules` are useless, don't need them anymore

`original_module_identifier`, `issuer` are moved to `ModuleFactoryCreateData`

`options` is moved to `NormalModuleFactory::new` when calling `new_compilation_params`